.github/workflows/ossf-scorecard-reporting.yml

```diff
@@ -17,15 +18,19 @@ jobs:
   security-scoring:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: OpenSSF Scorecard Monitor
-        uses: UlisesGascon/openssf-scorecard-monitor@v1.0.1
+        uses: UlisesGascon/openssf-scorecard-monitor@v2.0.0-beta1
```

Comment:
should we use beta?

Reply:
The beta is stable, I will do a separate PR in the future to move to `v2.0.0` once it is ready. The only changes expected from the beta to the final release is documentation related and additional unit testing